.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+### Reasons for making this change
+
+<!-- Add a reason for making this change here. -->
+
+### Related issues
+
+<!-- Add a reference to issues resolved, if applicable (for example, "fixes #1"). -->
+
+### Screenshots
+
+<!-- Add screenshots, if necessary -->
+
+### Checklist
+
+* [ ] I've added a screenshot of the changes, if this is a frontend change
+* [ ] I've added and/or updated tests, if this is a backend change
+* [ ] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
```

Comment:
I'm not sure if we need to explicitly add this, since the build will fail if style check fails.

Reply:
I don't think we have checks yet to make sure that the latest CLI / REST documentation is generated, though.